doc/nrf/security_chapter.rst

```diff
@@ -0,0 +1,59 @@
+Security in NCS
+###############
+
+Introduction
+************
+The following chapters gives a brief introduction to core security features available on Nordic products.
+The features are made available either as a built-in in modules, drivers and subsystems or as samples or application in nRF Connect SDK.
+
+
+Secure boot
+***********
+NCS supports secure boot of application images enforced by the bootloader.
+The secure boot utilizes signature validation and security hardware features to establish a root-of-trust during boot and to ensure the validity of the firmware that is booted.
+There are two available bootloaders in NCS, the `nRF Secure Immutable Bootloader (NSIB)<http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bootloader/README.html>` and a customized version of `MCUBoot <https://github.com/nrfconnect/sdk-mcuboot>`.
+To enable NSIB the configuration :kconfig:option:`CONFIG_SECURE_BOOT` need to be enabled.
+To enable MCUBoot the configuration :kconfig:option:`CONFING_MCUBOOT` need to be enabled.
+
+When enabling NSIB, it is possible to have an upgradable second-stage bootloader.
+
+Trusted Firmware-M
+******************
+The Trusted Firmware-M project (TF-M) is a reference design of the Arm Platform Security Arcitecture (PSA).
+NCS utilizes the security features of the Arm TrustZone techonolgy to configure secure peripherals and memory as well as to provide PSA functional APIs as secure services.
```

Comment:
When reading it, it seems to me that is should rather be "TF-M utilizes the ..." or perhaps "Through TF-M, NCS utilizes the ..."? Or maybe I'm misunderstanding?

Reply:
Through TF-M, NCS utilizes..." seem more clear to me as well. Done